src/lib/badge/badge.component.ts

```diff
@@ -3,7 +3,7 @@ import { Component, Input, ElementRef, ViewChild, OnChanges } from '@angular/cor
 @Component({
     selector: 'plex-badge',
     template: `
-        <span #badge class="badge badge-{{ type }} badge-{{ size }}">
+        <span #badge class="badge badge-{{ type }} badge-{{ size }} {{(size ? 'btn-' + size : '')}}">
```

Comment:
```suggestion
        <span #badge class="badge badge-{{ type }} badge-{{ size }} btn-{{ size }}">
```
Creo que no hace falta ese condicional, size tiene un valor 'md' por defecto, de hecho el badge usa el size sin codicional

Reply:
Ahi bien, muchas gracias Andrews!